functests/1_performance/cpu_management.go

```diff
@@ -153,6 +157,121 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 			table.Entry("Guaranteed POD should work on isolated cpu", true),
 		)
 	})
+
+	When("pod runs with the CPU load balancing runtime class", func() {
+		var testpod *corev1.Pod
+		var defaultFlags []int
+
+		getCPUSchedulingDomainFlags := func(cpu int) ([]int, error) {
+			cmd := []string{"/bin/bash", "-c", fmt.Sprintf("cat /proc/sys/kernel/sched_domain/cpu%d/domain*/flags", cpu)}
+			out, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
+			if err != nil {
+				return nil, err
+			}
+
+			var domainsFlags []int
+			for _, domainFlags := range strings.Split(out, "\n") {
+				flags, err := strconv.Atoi(domainFlags)
+				if err != nil {
+					return nil, err
+				}
+				domainsFlags = append(domainsFlags, flags)
+			}
+
+			sort.Ints(domainsFlags)
+			return domainsFlags, nil
+		}
+
+		BeforeEach(func() {
+			// get the default value for sched_domain flags, we will check the flags only for the CPU 0, because on the
+			// clean system it should be the same for all CPUs
+			var err error
+			defaultFlags, err = getCPUSchedulingDomainFlags(0)
+			Expect(err).ToNot(HaveOccurred())
+
+			testpod = pods.GetTestPod()
+			testpod.Namespace = testutils.NamespaceTesting
+
+			cpus := resource.MustParse("1")
+			memory := resource.MustParse("256Mi")
+
+			// change pod resource requirements, to change the pod QoS class to guaranteed
+			testpod.Spec.Containers[0].Resources = corev1.ResourceRequirements{
+				Limits: corev1.ResourceList{
+					corev1.ResourceCPU:    cpus,
+					corev1.ResourceMemory: memory,
+				},
+			}
+
+			// use the CPU load balancing runtime class
+			runtimeClassName := components.GetComponentName(profile.Name, components.ComponentNamePrefix)
+			testpod.Spec.RuntimeClassName = &runtimeClassName
+		})
+
+		AfterEach(func() {
+			// it possible that the VM already was deleted as part of the test, in this case we want to skip teardown
```

Comment:
VM -> pod? :)

Reply:
hh, looks like I thought about the KubeVirt, when I wrote it.